### PR TITLE
Add forms

### PR DESCRIPTION
**Pull request by Kelvin-Ben**
### Adding Forms the Blog App
This project had the following requirements:
- Creating a method called current_user in  ApplicationController to make current user data available to all controllers
   - It was to return the first user from the database.
- Created forms to perform the following functions:
   - Creates a Post on behalf of the current_user
   - Creates a comment on behalf of the current_user
- Allow Users to add likes to Posts.

### CODE REVIEW

File: app/controllers/application_controller.rb [hunk 1]
```diff
@@ -1,2 +1,7 @@
 class ApplicationController < ActionController::Base
+  # before_action :current_user
+  # @current_user ||= User.find_by(id: session[:user_id])
+  def current_user
```
Comment (arslanbisharat) [inline app/controllers/application_controller.rb:4]: - [ ] Kindly make sure there is no commented code present as it makes the code hard to read.

File: config/routes.rb [hunk 2]
```diff
@@ -1,12 +1,15 @@
...
   # Defines the root path route ("/")
   # root "articles#index"
-
   root 'users#index'
-
-  resources :users, only: [:index, :show] do
-    resources :posts, only: [:index, :show]
-  end
+  # get '/posts/new', to: 'posts#new'
+  # post '/posts', to: 'posts#create'
```
Comment (arslanbisharat) [inline config/routes.rb]: - [ ] Let's remove this commented code. It makes code unclean

File: db/seeds.rb [hunk 3]
```diff
@@ -5,3 +5,8 @@
 #
 #   movies = Movie.create([{ name: "Star Wars" }, { name: "Lord of the Rings" }])
 #   Character.create(name: "Luke", movie: movies.first)
```
Comment (arslanbisharat) [inline db/seeds.rb]: - [ ] Let's remove commented code.

**Review by arslanbisharat — CHANGES_REQUESTED** [review]
Hi @Kelvin-Ben, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights

- The Pull Request has a proper title and description. ✔️
- You have added a descriptive Readme file. Good Job. ✔️
- There is no linter error is present. ✔️
- Project has met all the technical aspects you needed for this project. Keep rocking! ✔️

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- [ ] Please make sure you right descriptive comments. Commenting involves placing Human Readable Descriptions inside of computer programs detailing what the Code is doing. Proper use of commenting can make code maintenance much easier, as well as helping make finding bugs faster. Further, commenting is very important when writing functions that other people will use. :x:


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: app/controllers/application_controller.rb (hunk 1 above)
Comment (Kelvin-Ben) [reply app/controllers/application_controller.rb:4]: Sure thing, Thank you, and consider it done

**Review by KayLemba — APPROVED** [review]
[# STATUS: APPROVED :green_circle: :clap: 

Hello  @Kelvin-Ben           , 
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: aa
Congratulations! 🎉 

HIGHLIGHTS:

- [x]  Your repo is professional 
- [x] Descriptive P.R
- [x] Tests are passing








 








Now you can merge this branch and move on :1st_place_medal: 
Happy Coding cheers 🥇 💯 



well-1

Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better.

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._]

**Pull request merged**